demo/references.d.ts

```diff
@@ -0,0 +1,2 @@
+/// <reference path="./node_modules/tns-platform-declarations/ios/ios.d.ts" />
+/// <reference path="./node_modules/nativescript-iqkeyboardmanager/IQKeyboardManager.d.ts" />
```

Comment:
New apps don’t have this `references.d.ts` file, so I _think_ this is supposed to be included in the `tsconfig.json` now.